dist/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,400;0,500;1,400&family=Roboto:wght@300;400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="css/main.css" />
+		<title>Authentication Page</title>
+	</head>
+	<body>
+		<nav class="navbar navbar-expand-lg navbar-light p-3 d-flex justify-content-center">
+			<span class="logo fw-bold fs-2">UserNest</span>
+		</nav>
+
+		<div class="container d-flex align-items-center vh-100">
+			<div class="row w-100">
+				<div class="col-lg-4 mx-auto">
+					<h2 class="text-center mb-4">Log in</h2>
+					<form action="feed/index.html" method="GET">
+						<div class="mb-3">
+							<label for="username" class="form-label">Username</label>
+							<input type="text" class="form-control" id="username" required />
```

Comment:
I guess you are to log in with your email and not your username. And if that is the case then the type should be email